### PR TITLE
Fix dependencies

### PR DESCRIPTION
**Pull request by ilyapuchka**
Please ensure that this type I didn't mess up with dependencies versions 😓

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -10,14 +10,16 @@ let package = Package(
...
-        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", from: "0.1.5"),
-        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .branch("master")),
+        .package(url: "https://github.com/jpsim/SourceKitten.git", .exact("0.20.0")),
+        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", .exact("0.1.5")),
+        .package(url: "https://github.com/kylef/Stencil.git", .exact("0.10.1")),
+        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .exact("2.4.0")),
         .package(url: "https://github.com/seanparsons/SwiftTryCatch.git", .revision("e7074a72e4d4dc516f391bc4d4afd8ca6a845b4b")),
-        .package(url: "https://github.com/xcodeswift/xcproj.git", from: "4.0.0"),
+        .package(url: "https://github.com/xcodeswift/xcproj.git", .exact("4.2.0")),
+        .package(url: "https://github.com/tadija/AEXML.git", .exact("4.2.2")),
```
Comment (ilyapuchka) [inline Package.swift]: had to add this because both SPM and CocoaPods were picking up 4.3.x version, which is Swift 4.1 only

File: Package.swift [hunk 2]
```diff
@@ -10,14 +10,16 @@ let package = Package(
...
         // PathKit needs to be exact to avoid a SwiftPM bug where dependency resolution takes a very long time.
         .package(url: "https://github.com/kylef/PathKit.git", .exact("0.8.0")),
-        .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.18.1"),
-        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", from: "0.1.5"),
-        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .branch("master")),
+        .package(url: "https://github.com/jpsim/SourceKitten.git", .exact("0.20.0")),
+        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", .exact("0.1.5")),
+        .package(url: "https://github.com/kylef/Stencil.git", .exact("0.10.1")),
+        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .exact("2.4.0")),
         .package(url: "https://github.com/seanparsons/SwiftTryCatch.git", .revision("e7074a72e4d4dc516f391bc4d4afd8ca6a845b4b")),
```
Comment (ilyapuchka) [inline Package.swift]: Not sure why this is different in Podfile, but we should probably stop using it at all

File: SourceryTests/Parsing/FileParser + MethodsSpec.swift [hunk 3]
```diff
@@ -29,7 +29,7 @@ class FileParserMethodsSpec: QuickSpec {
                     expect(methods[4]).to(equal(Method(name: "fooVoid()", selectorName: "fooVoid", returnTypeName: TypeName("Void"), definedInTypeName: TypeName("Foo"))))
                     expect(methods[5]).to(equal(Method(name: "fooInOut(some: Int, anotherSome: inout String)", selectorName: "fooInOut(some:anotherSome:)", parameters: [
                     MethodParameter(name: "some", typeName: TypeName("Int")),
-                    MethodParameter(name: "anotherSome", typeName: TypeName("inout String"), `inout`: true)
+                    MethodParameter(name: "anotherSome", typeName: TypeName("inout String"), isInout: true)
```
Comment (ilyapuchka) [inline SourceryTests/Parsing/FileParser + MethodsSpec.swift]: SwiftLint was messing with `inout` as parameter name, so I changed it for good

File: Package.swift [hunk 4]
```diff
@@ -10,14 +10,16 @@ let package = Package(
         .library(name: "SourceryJS", targets: ["SourceryJS"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/kylef/Commander.git", from: "0.6.1"),
```
Comment (Liquidsoul) [inline Package.swift]: I am seeing `0.6.0` for Commander in [Podfile.lock](https://github.com/krzysztofzablocki/Sourcery/blob/56b999afa87272bb9c70b907ff4e1922723ba91c/Podfile.lock#L3).

File: Package.swift [hunk 5]
```diff
@@ -10,14 +10,16 @@ let package = Package(
...
+        .package(url: "https://github.com/kylef/Commander.git", .exact("0.6.1")),
         // PathKit needs to be exact to avoid a SwiftPM bug where dependency resolution takes a very long time.
         .package(url: "https://github.com/kylef/PathKit.git", .exact("0.8.0")),
-        .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.18.1"),
-        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", from: "0.1.5"),
```
Comment (Liquidsoul) [inline Package.swift]: I am seeing `0.1.1` for libCommonCrypto in [Podfile.lock](https://github.com/krzysztofzablocki/Sourcery/blob/56b999afa87272bb9c70b907ff4e1922723ba91c/Podfile.lock#L4).

File: Package.swift [hunk 6]
```diff
@@ -10,14 +10,16 @@ let package = Package(
         .library(name: "SourceryJS", targets: ["SourceryJS"]),
     ],
     dependencies: [
-        .package(url: "https://github.com/kylef/Commander.git", from: "0.6.1"),
+        .package(url: "https://github.com/kylef/Commander.git", .exact("0.6.1")),
```
Comment (Liquidsoul) [inline Package.swift]: I am seeing `0.6.0` for Commander in [Podfile.lock](https://github.com/krzysztofzablocki/Sourcery/blob/56b999afa87272bb9c70b907ff4e1922723ba91c/Podfile.lock#L3).

File: Package.swift [hunk 7]
```diff
@@ -10,14 +10,16 @@ let package = Package(
...
         // PathKit needs to be exact to avoid a SwiftPM bug where dependency resolution takes a very long time.
         .package(url: "https://github.com/kylef/PathKit.git", .exact("0.8.0")),
-        .package(url: "https://github.com/jpsim/SourceKitten.git", from: "0.18.1"),
-        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", from: "0.1.5"),
-        .package(url: "https://github.com/SwiftGen/StencilSwiftKit.git", .branch("master")),
+        .package(url: "https://github.com/jpsim/SourceKitten.git", .exact("0.20.0")),
+        .package(url: "https://github.com/IBM-Swift/CommonCrypto.git", .exact("0.1.5")),
```
Comment (Liquidsoul) [inline Package.swift]: I am seeing `0.1.1` for CommonCrypto in [Podfile.lock](https://github.com/krzysztofzablocki/Sourcery/blob/56b999afa87272bb9c70b907ff4e1922723ba91c/Podfile.lock#L4).

File: Package.swift (hunk 6 above)
Comment (Liquidsoul) [reply Package.swift]: Checked, this difference is irrelevant, it just fixes a configuration issue in `Package.swift`.

File: Package.swift (hunk 7 above)
Comment (Liquidsoul) [reply Package.swift]: Ok, we are not using the same repositories, this is normal.

**Pull request merged**